quinn/src/send_stream.rs

```diff
@@ -134,6 +134,17 @@ impl SendStream {
     ///
     /// No new data may be written after calling this method. Completes when the peer has
     /// acknowledged all sent data, retransmitting data as needed.
+    ///
+    /// 0-rtt
+    /// ---
+    ///
+    /// Because 0-RTT packets cannot contain ACKs, if this is called from a 0-RTT stream opened by
+    /// the client, it cannot resolve for at least two round trips after the creation of the
+    /// connection.
```

Comment:
Can you briefly describe why "it cannot resolve for at least two round trips"?

Reply:
After further inspection of the RFCs, I realized I am confused about this bit. I will pick this back up when I understand better.

Reply:
I spent a while investigating this, and I was wrong about it. I was able to get it to resolve faster by creating a co-routine in the client that opens a new stream and writes a byte to it every millisecond, to trigger the server's ack threshold sooner. I think the moral is that `stopped` shouldn't be used for latency-sensitive things since acks can be delayed, so I added a note to that effect.

Reply:
Right, relying on transport-layer ACKs to drive application-layer behavior is almost always a bad idea.